script.js

```diff
@@ -1,111 +1,76 @@
+  add(data) {
+    if (data.title === '' || data.author === '') {
+      // eslint-disable-next-line no-alert
+      alert('Fields are empty');
+    } else {
+      const id = this.list.length + 1;
+      data.id = id;
+      this.list.push(data);
+      localStorage.setItem('data', JSON.stringify(this.list));
+      this.show(data);
+    }
   }
-};
-
-addBtn.addEventListener('click', (e) => {
-  addBook(e);
-  renderBooks();
-});
 
-const removeFromStorage = (book) => {
-  const bookTitle = book.querySelector('h3').textContent;
-  const bookAuthor = book.querySelector('p').textContent;
-  books = books.filter(
-    (book) => book.title !== bookTitle && book.author !== bookAuthor,
-  );
-  localStorage.setItem('books', JSON.stringify(books));
-};
-
-booksSection.addEventListener('click', (e) => {
-  if (e.target.classList.contains('remove-but')) {
-    e.target.parentElement.remove();
-    removeFromStorage(e.target.parentElement);
+  remove(r) {
+    const newData = this.list.filter((book) => book.id !== r);
+    this.list = newData;
+    localStorage.setItem('data', JSON.stringify(this.list));
+    this.getData();
   }
-});
 
-// class nameBook {
-//   constructor(name, writer){
-//     this.name = name;
-//     this.writer = writer;
-//   }
-//   theBook(name, writer){
-//     let albook = new Object();
-//     return (name + "by" + writer);
-//   }    
-// }
-// let newBooks = new nameBook(inputTitle, inputAuthor)
-// newBook.innerHTML = `
-//   <p>newBooks.thebook(inputTitle, inputAuthor)</<p
+  // eslint-disable-next-line class-methods-use-this
+  show(item) {
+    if (booksSection.innerHTML === 'No books added yet') {
+      booksSection.innerHTML = '';
+    }
+    const bookInfo = document.createElement('div');
+    bookInfo.classList.add('allBook');
+    const info = `
+    <div class="author-title">
+    <p class="title">${item.title}</p>
+    <p class="author">by ${item.author}</p>
+    </div>
+    <button class="remove-but" onclick="newbooks.remove(${item.id})" type="button" id="book-${item.id}">Remove</button>
+  `;
+    bookInfo.innerHTML = info;
+    booksSection.appendChild(bookInfo);
+  }
 
-// class bookText {
-//   static bookTest = [];
-//   static #booksSection = document.querySelector('bookTest');
-//   constructor(title, author){
-//     this.title = title;
-//     this.author = author;
-//   }  
-// }
+  getData() {
+    if (this.list.length > 0) {
+      booksSection.innerHTML = '';
+      this.list.forEach((book) => {
+        this.show(book);
+      });
+    } else {
+      booksSection.innerHTML = 'No books added yet';
+    }
+  }
+}
 
-// add(){
-//   bookText.bookTest.push(this);
-// }
+this.addBut = document.querySelector('#addBut');
+this.addBut.addEventListener('click', (e) => {
+  e.preventDefault();
+  const book = new First();
+  book.title = document.getElementById('title').value;
+  book.author = document.getElementById('author').value;
+  // eslint-disable-next-line no-use-before-define
```

Comment:
Kindly remove this line of code and fix the issue flagged by the linters tool. :pray: :pray: